src/test/java/step_definitions/WikipediaSteps.java

```diff
@@ -24,9 +25,9 @@
     public WikipediaSteps() throws Exception {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AS00112)